_config.yaml

```diff
@@ -31,9 +31,12 @@ theme: minima
 header_pages:
   # Note that when first cloned, the `workshop` directory will not exist,
   # as described at https://github.com/AlexsLemonade/training-specific-template#customizing-the-new-repository-for-an-individual-training-workshop
+  - workshop/HOME.md
```

Comment:
I think I can make this better (have the main head point here). Will be in a separate PR.

Reply:
See https://github.com/AlexsLemonade/2020-june-training/pull/6